tests/unit/StaffNotation.spec.ts

```diff
@@ -503,6 +505,10 @@ describe('StaffNotation', () => {
     expect(buildRestorationFiling).toHaveBeenCalled()
     expect(createFiling).toHaveBeenCalled()
 
+    // need to update DOM to not get previous test results
+    await Vue.nextTick()
+    await Vue.nextTick()
```

Comment:
I was getting some weird behavior here. This test was returning the value of the previous one. The next test was returning the value of this test. This fixes that by waiting for the DOM to update. Ditto lines 563, 564, and 565.

Keep in mind I've tested this in local and it works fine!

Reply:
Yes, we've seen this elsewhere. There's a test function kicking around to do "await nextTick" and/or "await flushPromises" a few times for this purpose.

Why? Possibly some things that take more than 1 tick to complete, like validation maybe, or transitions.

(1 tick = 1 timeslice = 1 flow of code at a time = 1 message from event-driven architecture, which this is) Ref: [link](https://en.wikipedia.org/wiki/JavaScript#:~:text=event%2Ddriven%2C%20functional%2C%20and%20imperative%20programming%20styles)

Reply:
LGTM